examples/gocean/eg6/Makefile

```diff
@@ -0,0 +1,75 @@
+#------------------------------------------------------------------------------
+# BSD 2-Clause License
+# 
+# Copyright (c) 2020, Science and Technology Facilities Council.
+# All rights reserved.
+# 
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+# 
+# * Redistributions of source code must retain the above copyright notice, this
+#   list of conditions and the following disclaimer.
+# 
+# * Redistributions in binary form must reproduce the above copyright notice,
+#   this list of conditions and the following disclaimer in the documentation
+#   and/or other materials provided with the distribution.
+# 
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+# DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+# FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+# DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+# SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+# CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+# OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#------------------------------------------------------------------------------
+# Authors: J. Henrichs, Australian Bureau of Meteorology
+
+F90 ?= gfortran
+F90FLAGS ?= -g -O0
+
+DL_DIR = ../../../external/dl_esm_inf/finite_difference/src
+EXTRACT_DIR = ../../../lib/extract/dl_esm_inf/netcdf
+F90FLAGS += $$(nc-config --fflags) -I$(DL_DIR)
+LIB_NAME = lib_kernel_data_netcdf.a
+
+
+KERNELS = init_field_mod.o update_field_mod.o
+EXEC = extract_test
+DRIVER_INIT = driver-main-init
+
+all:	$(EXEC) $(DRIVER_INIT)
+
+test:	$(DRIVER_INIT)
+
+$(DRIVER_INIT):	$(KERNELS) $(DRIVER_INIT).o main.o \
```

Comment:
Aeh yes, good point. The real problem is actually that the module/region name created after Rupert's latest update contain a : ... and I can't create a file with ":" in its name on my window-filesystem that I mount on my linux VM. In #655 I actually properly support renaming the extract regions (based on the profiling code), but anything based on PSyData. So the Makefile contained these new names by mistake (but even if it would use the right filenames it wouldn't compile). So I've removed the driver compilation from the default target, and added a reference to #644.

Reply:
Great, thanks.